DAQ_System/Source/Adapter/Interfaces/ilinear_potentiometer.hpp

```diff
@@ -0,0 +1,28 @@
+/*
+* Linear Potentiometer
```

Comment:
Thoughts on adding the part/model of the hardware component? I'm thinking `SLS1300 Linear Potentiometer`

Reply:
added in recent push